.golangci.yml

```diff
@@ -18,6 +22,22 @@ linters-settings:
       - prefix(github.com/aws/eks-anywhere)
     custom-order: true
     skip-generated: false
+  godot:
+    exclude:
+      # Exclude todo comments.
+      - "^todo:"
+    period: true
+    capital: true
+  nakedret:
+    # never allow naked returns
+    max-func-lines: 0
+  gocyclo:
+    # Minimal code complexity to report.
+    min-complexity: 10
```

Comment:
Curious if you did any testing against this? I haven't evaluated the algorithm for gocyclo but Go typically suffers a little because of how expressive it is and consequently you often need to bump by 1 or 2 points.

Reply:
Nop, I didn't. 10-20 is the recommended value by gocyclo, so I opted for started with the lowest limit and go from there. Making it less restrictive is easier than making it more restrictive.